### ISSUE TITLE
0.4.0: The wordlist file does not exist

### ISSUE THREAD

**Issue by blshkv** [maurosoria/dirsearch#650]
Hi, the previous version 0.3.9 was working fine.

I have upgraded to the latest and facing this issue:
```
dirsearch -u https://xxx.com -e php
The wordlist is invalid
```

ps. The tool is installed as a system package, i.e. the file location is the following:
/usr/share/dirsearch/db/dicc.txt

**Comment by shelld3v** [comment]
Hi, can you try this command: `git clone https://github.com/maurosoria/dirsearch.git` to install with the latest commit? If the bug is still there, tell me and I will fix it before release dirsearch v0.4.1, so your tool can run dirsearch perfectly after my release =) (If the bug is still there and you don't tell me, the bug will be still in the next version and you will need to wait after releasing v0.4.2 to can work with your tool!)

**Comment by blshkv** [comment]
ok, it is fixed in the latest commit. Please release at some point
